app/js/editQuestion.js

```diff
@@ -57,52 +47,9 @@ window.addEventListener("load", async () => {
     document.querySelector(`input[data-question-id="${answer}"]`).checked = 1;
 
     form.addEventListener("submit", async (event) => {
-       event.preventDefault();
-
-       const responseMsg = document.querySelector('#inputSubmissionConfirmation');
-       const hasQuestions = formHasQuestion(form);
-       const hasAnswers = formHasBetweenTwoAndFiveAnswers(form);
-       const isValid = answerHasValidValue(form);
-
-        if (hasQuestions && hasAnswers && isValid) {
-            let answer;
-
-            if (!authorised) {
-                return;
-            }
-
-            document.querySelectorAll('.answer-check').forEach((answerCheck, index) => {
-                if (answerCheck.checked) {
-                    answer = index + 1;
-                }
-            });
-
-            const { success, message } = await editQuestion(questionId,{
-                text: document.querySelector("#question-to-edit").value,
-                option1: document.querySelector("#option1").value,
-                option2: document.querySelector("#option2").value,
-                option3: document.querySelector("#option3").value,
-                option4: document.querySelector("#option4").value,
-                option5: document.querySelector("#option5").value,
-                test_id: document.querySelector("#test_id").value,
-                answer
-            });
-
-            responseMsg.textContent = message;
-
-            if (success){
-                responseMsg.classList.add('alert-success');
-                responseMsg.classList.remove('alert-danger');
-            } else {
-                responseMsg.classList.remove('alert-success');
-                responseMsg.classList.add('alert-danger');
-            }
-        } else {
-            responseMsg.classList.remove('alert-success');
-            responseMsg.classList.add('alert-danger');
-            responseMsg.textContent = 'Error: Please ensure you have filled out the question form correctly.';
-        }
+        await sendQuestionForm(event, form, true, questionId);
```

Comment:
why are you awaiting this? theres no code after this so theres nothing to wait for?